src/setupTests.ts

```diff
@@ -3,3 +3,6 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect';
+
+jest.spyOn(window, 'fetch').mockImplementation(
```

Comment:
By using `jest.spyOn()` we can mock functionality of existing modules (in this case `fetch`). This ensures that we don't use the actual functionality, which in this case would attempt to send an HTTP request. By mocking `fetch` we can modify its behavior and pretend that HTTP requests are sent out, although everything is just happening locally. See the `useApi.test.ts` for examples on how to make this work.